src/lib/airtable/request.js

```diff
@@ -646,6 +667,23 @@ export const updateManyInventorys = async (recordUpdates) => {
   return Promise.all(updatePromises);
 };
 
+// NONGENERATED: Approve/deny purchase request and update inventory qty if approved
+export const updatePurchaseRequestAndInventory = async (purchaseRequest) => {
```

Comment:
This is now just `updatePurchaseRequest` right? Since inventory update is in `createPurchaseRequest`